scripts/jenkins/fhe_docker_jenkins_trigger_builds_centos.sh

```diff
@@ -31,7 +31,17 @@ set -e
 git checkout master
 # Build the Docker image for CentOS
 ./BuildDockerImage.sh centos
+# Shut everything down before we start
+./StopToolkit.sh
 # Run the toolkit container based on the CentOS image
 ./RunToolkit.sh -l -s centos
+# Test sample runs as expected
+docker exec local-fhe-toolkit-centos /bin/bash -c " \
+    cd /opt/IBM/FHE-Workspace; \
+    mkdir build; cd build; \
+    cmake ../examples/BGV_country_db_lookup;\
+    make;\
+    cd ..;\
+    ./examples/BGV_country_db_lookup/runtest.sh;"
 # Shut everything down 
-./StopToolkit.sh
\ No newline at end of file
+./StopToolkit.sh
```

Comment:
There needs to be a command added at the end that removes the docker container from the server otherwise the server will fill up.

I think something as simple as `docker prune --all` is probably good enough, but there may be something more specific to each build that we can use instead.  Although if that is added to the end of each script it will only have to remove one thing each time anyway

Reply:
Thanks Greg. I had thought stopping and removing the container would be enough.

Reply:
haha well I only know this because this was a task I was supposed to do last time we filled up the HD space and I forgot.  So I figured while you're in there might as well...  thank you for doing my job